solutions/task_4.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+"""
+TASK: Implement classes "Temperature", "Person" using Python descriptor protocols to execute unittests without errors.
+"""
+import re
+import unittest
+from datetime import datetime
+
+
+class Temperature(object):
+
+    def __init__(self, value=0):
```

Comment:
Why are you naming variables so common ```value```? :((((((
Is ```fahrenheit```bad name? Long? Ugly? Not clear? Is ```value``` more clear?

Reply:
Ok